aplib.net-demo/Assets/Scripts/Entities/Player/Movement.cs

```diff
@@ -1,5 +1,9 @@
 using UnityEngine;
 
+[RequireComponent(typeof(Rigidbody))]
+[RequireComponent(typeof(CapsuleCollider))]
+[RequireComponent(typeof(EntitySound))]
```

Comment:
Don't think you want to do sound stuff here either. Though currently we don't have a Move event.

Reply:
This will be replaced when animation clips get added, as discussed irl